core-bundle/src/Command/SuperviseWorkersCommand.php

```diff
@@ -49,6 +49,12 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new SymfonyStyle($input, $output);
 
+        if (!$this->workers || !$this->supervisor) {
```

Comment:
Actually, I don't really think it makes sense to have the entire command if the only thing it does is display an error in such a case?

Reply:
Well yes, that's why you remove it from the container. So in reality, this will never occur - but technically those 2 constructor arguments need to be nullable.

Reply:
I disagree. Calling `new SuperviseWorkersCommand` without those arguments never makes sense. A class has to be built so that the requirements are part of the constructor which they currently are.

Reply:
Then we will need to adjust our `ServiceArgumentsTest` accordingly instead.

Reply:
But if we used autowiring, we would have to make the arguments nullable as well, wouldn't we?

Reply:
? How is this related to autowiring at all? This command cannot exist without a `Supervisor` instance or workers.

Reply:
Then the parameters should not be nullable I guess.

Reply:
I checked the test and indeed we do not allow `null` in the configuration if the parameter is not nullable:

https://github.com/contao/contao/blob/fd1115977665d47af4ea0554808950a85bcae6f4/core-bundle/tests/Functional/ServiceArgumentsTest.php#L93-L95

So to fix this, we have to set something other than `null` here:

<img width="471" alt="" src="https://github.com/user-attachments/assets/f82ca3e2-b16c-45d4-ab75-411032428587" />

We have the same problem with the `contao.security.login_authenticator` service, which was also not noticed because the service is not part of the test container.

Reply:
@fritzmg Will you complete this PR or shall we close it?

Reply:
What do I need to change in this PR?